plutus-core-spec/main.tex

```diff
@@ -699,7 +703,7 @@
 %
 % paper title
 % can use linebreaks \\ within to get better formatting as desired
-\title{Formal Specification of\\the Plutus Core Language v1.0 (RC5.3)}
+\title{Formal Specification of\\the Plutus Core Language v1.0 (RC5.4)}
```

Comment:
I've seen a RC5.4 which was different from this one. Can we jump straight to 5.5?

Reply:
Done.